doc/ug/design.md

```diff
@@ -30,6 +30,11 @@ TODO: briefly discuss key architectural decisions, and how we came to the conclu
 *   Bus strategy
 *   Reset strategy
 
+## Secure Hardware Design Guidelines
+
+Silicon designs for security devices require special guidelines to protect the designs against myriad attacks.
+To that end, the team established [Secure Hardware Design Guidelines]({{< relref "doc/security/implementation_guidelines/hardware" >}}) which are followed when developing OpenTitan security IP.
+
```

Comment:
Is it desirable to have an exact copy of the section you added in `doc/security/_index.md` here?

Reply:
The idea was to make this page better discoverable and this page here is the overview page that has several security related pages underneath (in the folder structure).

When you go to this page today (https://docs.opentitan.org/doc/security/) it is a bit odd to see a link to "Implementation Guidelines" on the left side, but no associated section on the overview page itself.

Reply:
Maybe add it as a link to the `Language and Tool Selection` section instead of making it it's own section?

Reply:
Also, maybe we can add a link on this page as well? https://docs.opentitan.org/doc/ug/hw_design/

Reply:
OK, amended accordingly - PTAL.